docs/content/routing/providers/kubernetes-crd.md

```diff
@@ -1660,16 +1660,13 @@ or referencing TLS stores in the [`IngressRoute`](#kind-ingressroute) / [`Ingres
     
     spec:
       entryPoints:
-        - web
+        - websecure
```

Comment:
Indeed, it would make more sense to attach to the `websecure` entryPoint, or more demonstrative. But it is not required, as nothing prevents attaching a TLS router on any port. I'm just wondering whether it was done on purpose.

Reply:
Me too :). If it was done on purpose, then maybe an explanation would help.